myokit/_sim/cvodessim.c

```diff
@@ -195,11 +195,19 @@ Model model;        /* A model object */
 /*
  * Pacing
  */
-ESys epacing;           /* Event-based pacing system */
-PyObject* eprotocol;    /* An event-based pacing protocol */
-FSys fpacing;           /* Fixed-form pacing system */
-PyObject* fprotocol;    /* A fixed-form pacing protocol */
-double pace = 0;        /* Pacing value */
+union PSys {
+    ESys event;
+    FSys fixed;
+};
+enum PSysType {
+    EVENT,
+    FIXED
+};
+union PSys *pacing_systems;   /* array of pacing system (event or fixed) */
+enum PSysType *pacing_types; /* array of pacing system types */
```

Comment:
Not sure I understand the syntax of repeating the union and enum keywords here...

Reply:
the joy of C.....